src/main/java/spring/data/access/Main.java

```diff
@@ -1,8 +1,27 @@
 package spring.data.access;
 
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import spring.data.access.entity.Cart;
+import spring.data.access.entity.Product;
+import spring.data.access.repository.JDBCTemplateCart;
+import spring.data.access.repository.JDBCTemplateProduct;
+
 public class Main {
 
     public static void main(String[] args) {
+        AnnotationConfigApplicationContext applicationContext
+                = new AnnotationConfigApplicationContext("spring.data.access");
+
+        JDBCTemplateProduct jdbcTemplateProduct = applicationContext.getBean(JDBCTemplateProduct.class);
```

Comment:
here you must work with Services not Repositories / DAO

Reply:
have been changed to work with services